.github/workflows/ci.yml

```diff
@@ -150,6 +150,8 @@ jobs:
         run: make load-design-system-templates
       - name: collectstatic
         run: poetry run ./manage.py collectstatic --verbosity 0 --noinput --clear
+      - name: compilemessages
+        run: poetry run ./manage.py compilemessages --verbosity 0 --use-fuzzy
```

Comment:
I don't recommend using strings marked as fuzzy in production. These are entries where the translator wasn’t fully confident. Ours exists because @MaciekBaron added a bunch for development purposes. It's not a big deal if the translations aren’t perfect right now, since a native Welsh speaker will review and finalise them before launch.

Reply:
For now, worth dropping `fuzzy` from PO file so have as much in Welsh as possible. They will all be re-reviewed anyway, so even if the translfation is 100% correct, it's fine for now.

Reply:
Fixed in https://github.com/ONSdigital/dis-wagtail/pull/246/commits/ccb7874f64177380d0136eb7aac848b247ef0446

Reply:
I think that may need removing the related msgids too. 

https://stackoverflow.com/questions/2710733/djangos-makemessages-creates-a-lot-of-fuzzy-entries has the tool to use.

The diff should be like:

```diff
- #, fuzzy
- #~| msgid "Release date"
- #~ msgid "Releases:"
- #~ msgstr "Dyddiad y datganiad"
```